src/foam/u2/layout/AlignmentTypes.js

```diff
@@ -0,0 +1,24 @@
+foam.ENUM({
+  package: 'foam.u2.layout',
+  name: 'AlignmentTypes',
+
+  documentation: `
+      An enum for the alignment types of a ~row of columns~
```

Comment:
If it's just for columns then the name `AlignmentTypes` should reflect that. That said, this enum should be apply to apply to any of our layouts so I don't think you need to rename this or call this out as for a row of columns in the documentation. Documentation could just be something like: Alignment types for the layout package.

Reply:
Done